### ISSUE TITLE
Nan, cellEdited

### ISSUE THREAD

**Issue by DimasAda** [olifolkerd/tabulator#2412]
sorry for my english. 
 ```
 var table = new Tabulator('#example-table', {
 columns:[
                   {title:"Time", field:"Time", align:"center", editor: "number"},
                   {title:"Plan", field:"Plan", align:"center", editor: "number"},
                   {title:"Plan2", field:"Plan2", align:"center",
                   mutator:function(value, data){
                     return (data.Plan * data.Time).toFixed(2);}
                  },
                    ],
                  cellEdited:function(cell){
                  cell.getRow().update({Plan2:false});
                      },
        });
        $("#file-load-trigger").click(function(){
        table.setDataFromLocalFile();
        });

```
1. Step # 1 - loading a table from a file. The mutator calculated correctly.
2. Step #  2 - cell change. The mutator calculated NaN.

**Comment by DimasAda** [comment]
sorry for my english. 
 ```
 var table = new Tabulator('#example-table', {
 columns:[
                   {title:"Time", field:"Time", align:"center", editor: "number"},
                   {title:"Plan", field:"Plan", align:"center", editor: "number"},
                   {title:"Plan2", field:"Plan2", align:"center",
                   mutator:function(value, data){
                     return (data.Plan * data.Time).toFixed(2);}
                  },
                    ],
                  cellEdited:function(cell){
                  cell.getRow().update({Plan2:false});
                      },
        });
        $("#file-load-trigger").click(function(){
        table.setDataFromLocalFile();
        });

```
1. Step # 1 - loading a table from a file. The mutator calculated correctly.
2. Step #  2 - cell change. The mutator calculated NaN.

**Comment by olifolkerd** [comment]
Hey @DimasAda,

Yes this is because at present only the edited data fields are passed to the mutator but i can see this being an issue so i will resolve it for you in the 4.5 release.

Thanks for letting me know.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @DimasAda 

This has now been resolved in the 4.6 branch, which will be released later this weekend.

Cheers

Oli :)
